src/mudlet-lua/lua/geyser/GeyserMiniConsole.lua

```diff
@@ -296,6 +296,19 @@ function Geyser.MiniConsole:resetAutoWrap()
   setWindowWrap(self.name, self.wrapAt)
 end
 
+--- The same as Mudlet's base display(), but outputs to the miniconsole instead of the main window.
+function Geyser.MiniConsole:display(...)
+  local arg = {...}
+  arg.n = table.maxn(arg)
```

Comment:
This isn't safe if the table has holes - try our `table.size` instead.

Reply:
Should I update the original display() function as well then? I just copied it across.

Reply:
Yeah, please.

Reply:
actually, I think table.maxn might be the right thing here

![image](https://user-images.githubusercontent.com/3660/88460338-87981380-ce69-11ea-8d54-7a30fa84e871.png)

Reply:
@vadi2 thoughts?

Reply:
Let's stick with this then